local_nav_pkg/Corner box
float32 confidence
int32 type
